packages/react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -175,7 +175,9 @@ Checkbox.create = createShorthandFactory({
 })
 
 /**
- * A single checkbox within a checkbox group.
+ * A Checkbox allows a user to make choice from a small set of options.
```

Comment:
`small set of options` sounds confusing in my opinion. Can we be more specific with this, or at least state dual-state for now, until we actually support more options

Reply:
replaced with direct `two options`, as this is the only mode we currently support